packages/gatsby-theme-styleguide/src/gatsby-components/Menu/Menu.js

```diff
@@ -169,24 +203,27 @@ class Menu extends Component {
         query={query}
         render={data => {
           return (
-            <Container>
-              <MenuHeader
-                siteTitle={siteTitle}
-                githubReleases={data.allGithubRelease.edges}
-              />
-              <NavContainer>
-                {this.buildMenu(this.state.menuArray, true)}
-
-                <DesignerKitLink
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="button__menu button button--secondary"
-                  title="Download the IU kit"
-                >
-                  Download the design kit
-                </DesignerKitLink>
-              </NavContainer>
-            </Container>
+            <React.Fragment>
```

Comment:
tips : instead of `<React.Fragment>... </React.Fragment>` you can just do `<>...</>`

Reply:
fixed